schema.sql

```diff
@@ -61,4 +61,20 @@ CREATE TABLE visits (
     visit_date DATE NOT NULL,
     CONSTRAINT fk_animal_id FOREIGN KEY(animal_id) REFERENCES animals(id),
     CONSTRAINT fk_vets_id FOREIGN KEY(vet_id) REFERENCES vets(id)
-);
\ No newline at end of file
+);
+
+-- SECOND WEEK (Pair-programming)
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+
+-- This will add 3.594.280 visits considering you have 10 animals, 4 vets, and it will use around ~87.000 timestamps (~4min approx.)
+INSERT INTO visits (animal_id, vet_id, date_of_visit) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
+
+-- This will add 2.500.000 owners with full_name = 'Owner <X>' and email = 'owner_<X>@email.com' (~2min approx.)
+insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
```

Comment:
- [ ] These `INSERT INTO` querries are supposed to be in the `data.sql` file. Please remove them here and add them in the proper file which is `data.sql` file 😃

Reply:
Done ✔️